components/infoColumns/index.tsx

```diff
@@ -98,57 +98,65 @@ const InfoColumns = () => {
           post-graduates, and professionals from backgrounds in academia,
           finance, politics, business, technology and more to deliver a
           high-caliber educational experience to our delegates.
+          <br />
+          <br />
+          This coming year, we welcome you - delegates, faculty, sponsors, and all others -
+          <em> in-person</em>, to the <strong> 18th Chicago International Model United Nations
+          conference</strong>. We hope you can make it!
         </Body>
       </CardContainer>
-      <CardWrapper flex="2" margins="1rem 0" pad="1rem 0 1rem 1rem">
+      <CardWrapper flex="2" margins="1rem 0" pad="1rem 0 6rem 1rem">
         <CardContainer
           bordered
           height="600px"
-          color={colors.fadedPrimaryBlue}
+          color={colors.primaryBlue}
           margins="1rem 1rem 0 0"
         >
-          <SubTitle size="2.5rem" margins="20px 0" self="center">
-            Join Us!
+          <SubTitle size="3.5rem" margins="10px 0" self="center" align="center" line={0.9} color={colors.ltGray}>
+            Join us in-person!
           </SubTitle>
           <div
             style={{
               alignItems: "center",
               justifyContent: "center",
             }}
           >
-            <Body align="center" self="center" size="1.2rem">
-              <b>February 3rd - 6th 2022</b>
+            <Body align="center" self="center" size="2rem" weight={700} color={colors.ltGray}>
+              February 3rd - 4th, 2022
             </Body>
-            <Body align="center" self="center" size="1.2rem">
+            <Body align="center" self="center" size="2rem" color={colors.ltGray}>
               <b>The Palmer House Hilton Hotel</b>
               <br />
-              <i>17 East Monroe Street Chicago, IL 60603</i>
+              <i>17 East Monroe Street<br />Chicago, IL 60603</i>
             </Body>
           </div>
         </CardContainer>
         <CardContainer
           bordered
           height="600px"
-          color={colors.fadedltGray}
+          color={colors.fadedPrimaryBlue}
           margins="1rem 0 0 1rem"
         >
-          <SubTitle size="2.5rem" margins="20px 0" self="center" align="center">
+          <SubTitle size="3.5rem" margins="10px 0" self="center" align="center" line={0.9}>
```

Comment:
Same breakpoint refactor here to have this subtitle match the Title styled component